### PR TITLE
Article edit separate

### PR DESCRIPTION
**Pull request by vinyll**
We had an edit-in-place.
Now we have a separated page instead.

### CODE REVIEW

File: user/views.py [hunk 1]
```diff
@@ -45,10 +45,17 @@ def privacy_delete_trip(date):
...
 
 
-@app.route('/profile/', methods=['patch'])
+@app.route('/profile/')
 @login_required
 def profile_edit():
-    data = request.json
+    user = User.objects.get(id=current_user.id)
+    articles = Article.objects(author=user)
+    return render_template('user/edit.html', user=user, articles=articles)
```
Comment (fuentesloic) [inline user/views.py:53]: template `user/edit.html` doesn't exist

File: templates/user/profile.html [hunk 2]
```diff
@@ -50,182 +52,32 @@ <h1 name=username>{{ user.username or 'Insert a username' }}</h1>
     {% endfor %}
     </section>
     <nav>
```
Comment (fuentesloic) [inline templates/user/profile.html:54]: you have just a link so should not we remove the `<nav>` ?

File: templates/user/profile.html [hunk 3]
```diff
@@ -50,182 +52,32 @@ <h1 name=username>{{ user.username or 'Insert a username' }}</h1>
...
-      {% if edit %}
-      <a class="action-button settings" href={{ url_for('privacy') }}>Set my privacy settings</a>
-      <a class="action-button trips" href={{ url_for_trips(user) }}>View my trips</a>
-      {% else %}
-      <a class="action-button trips" href={{ url_for_trips(user) }}>View {{ user }}'s trips</a>
+      {% if user == current_user %}
+        <a class=button href={{ url_for('profile_edit') }}>Edit my profile</a>
```
Comment (fuentesloic) [inline templates/user/profile.html:56]: you should user class`action-button` to keep a consistant style (try to keep ^^)

File: templates/user/profile.html [hunk 4]
```diff
@@ -27,16 +27,18 @@
...
         {{ user }} is currently around <a href={{ url_for_trips(user) }} data-latlng={{ user.current_location and user.current_location|join(',') }}>the world</a>.</p>
       {% else %}
-      <a class=private href={{ url_for('privacy') }}>
-        Community mode is disabled.<br>
-        your contents are restricted and private.<br>
-      </a>
+        <div class=private>
+          {{ user }} disabled community mode; the contents are therefore private.<br>
+          {% if user == current_user %}
+            <a href={{ url_for('privacy') }}>Change my settings</a>
```
Comment (fuentesloic) [inline templates/user/profile.html:39]: same here add class `action-button`

File: templates/user/profile.html [hunk 5]
```diff
@@ -27,16 +27,18 @@
...
       {% if user.allow_community %}
       <p>
         {{ user }} is currently around <a href={{ url_for_trips(user) }} data-latlng={{ user.current_location and user.current_location|join(',') }}>the world</a>.</p>
       {% else %}
-      <a class=private href={{ url_for('privacy') }}>
-        Community mode is disabled.<br>
-        your contents are restricted and private.<br>
-      </a>
+        <div class=private>
+          {{ user }} disabled community mode; the contents are therefore private.<br>
```
Comment (fuentesloic) [inline templates/user/profile.html:37]: remove the `<br>` use `display:flex + flex-direction: column` instead

File: templates/user/profile.html [hunk 6]
```diff
@@ -27,16 +27,18 @@
         onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'">
     </figure>
     <div>
-      <h1 name=username>{{ user.username or 'Insert a username' }}</h1>
-      <p name=about>{{ user.about|htmlnewline or 'Introduce yourself here' }}</p>
+      <h1 name=username>{{ user.username }}</h1>
+      <p name=about>{{ user.about|htmlnewline }}</p>
       {% if user.allow_community %}
       <p>
         {{ user }} is currently around <a href={{ url_for_trips(user) }} data-latlng={{ user.current_location and user.current_location|join(',') }}>the world</a>.</p>
```
Comment (fuentesloic) [inline templates/user/profile.html:34]: the `</p>` should be go to a next line

Comment (fuentesloic) [reply templates/user/profile.html:34]: even the `<a>...</a>` the line is suuuuper long

**Review by fuentesloic — CHANGES_REQUESTED** [review]

**Comment by vinyll** [comment]
Well, actually the whole frontend is pretty much broken.
I'm trying to clean and reset the server. Do you think you could continue this PR with the frontend @fuentesloic ?

**Comment by fuentesloic** [comment]
@vinyll no prob (I hope this weekend)

**Comment by vinyll** [comment]
This was replaced by #258

**Pull request closed without merging**